examples/test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from buffling import privacy
+import numpy as np
+import pandas as pd
+
+
+# global parameters
+EPSILON = np.log(3)
+DELTA = 1e-4
+NUM_PUB = 8
+NUM_BIT = 1000000
+NUM_SAMPLING = 500000
+
+
+def main():
+  # required p for one BF
+  print(f"epsilon = {EPSILON}, delta = {DELTA}\n")
```

Comment:
Print out information when running the code is nice. Do you prefer to use logging of absl? The logging module is powerful, and allows you to log your output in other format. For example, you can choose to print out the message in commandline, and/or save to a designated file location, etc.

An example: (line 17)
https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/run_tests.py

Reply:
absl sounds great! I will use it instead.